packages/yuudachi/src/functions/autocomplete/reasons.ts

```diff
@@ -1,14 +1,9 @@
 import type { AutocompleteInteraction } from "discord.js";
 import i18next from "i18next";
-import { AUTOCOMPLETE_REASON_FOCUSED_FIELD_NAME } from "../../Constants.js";
 import { findBestMatch } from "../../util/findBestMatch.js";
 
-export function checkReasonAutocomplete(interaction: AutocompleteInteraction) {
-	return interaction.options.getFocused(true).name === AUTOCOMPLETE_REASON_FOCUSED_FIELD_NAME;
-}
-
-export async function handleAutocompleteReasons(interaction: AutocompleteInteraction, locale: string): Promise<void> {
-	const input = interaction.options.getString(AUTOCOMPLETE_REASON_FOCUSED_FIELD_NAME, true);
+export async function handleReasonAutoComplete(interaction: AutocompleteInteraction, locale: string): Promise<void> {
```

Comment:
```suggestion
export async function handleReasonAutocomplete(interaction: AutocompleteInteraction, locale: string): Promise<void> {
```

Change everywhere it's used too.